TF-15131 Update product documentation with new default project behavior (#689)
hashicorp/terraform-docs-common: website/docs/cloud-docs/projects/managing.mdx

1. Click on a project name of the project you want to edit.
1. Click **Settings**.

On this **General settings** page, you can update the project name and project
description, override the default project, and delete the project. On the **Team
access** page, you can modify team access to the project.

## Delete a Project

> You can only delete a project that does not contain workspaces and is not the

Reviewer: I think it's a little clearer to start with the default project in this sentence like so:
```suggestion
You cannot delete the default project or projects that contain workspaces.
```

> default project.

Author: In the future, we will allow deleting a default project but that isn't the case today.
